libs/ha/ha_common_libs_k8s.py

```diff
@@ -741,22 +741,26 @@ def poll_cluster_status(self, pod_obj, timeout=1200):         # default 20mins t
         LOGGER.debug("Time taken by cluster restart is %s seconds", int(time.time()) - start_time)
         return resp
 
-    def restore_pod(self, pod_obj, restore_method, restore_params: dict = None, clstr_status=False):
+    def restore_pod(self, pod_obj, restore_method, restore_params: dict = None,
+                    clstr_status=False, num_replica=1, set_name=None):
         """
         Helper function to restore pod based on way_to_restore
         :param pod_obj: Object of master node
         :param restore_method: Restore method to be used depending on shutdown method
         ("scale_replicas", "k8s", "helm")
         :param restore_params: Dict which has parameters required to restore pods
-        :clstr_status: Flag to check cluster status after pod restored
+        :param clstr_status: Flag to check cluster status after pod restored
```

Comment:
can you add doc for num_replica and set_name?

Reply:
@ApurwaMohite  why did u remove num_replica and set_name from args or u wanted to add *kargs???

Reply:
Those params are added in restore_params dict which we pass from test. So removed from here.